calblueprint/girls-write-now
# [setting] Update setting style

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Fixed styling of heading and subheading text in the settings page (this was already done by a previous PR)
- Made the bottom tabs display when the user is on the settings page 

## Relevant Links

### Notion Sprint Task

[//]: # 'Add the relevant Notion link(s) here'

https://www.notion.so/calblueprint/setting-Update-setting-style-9213460b54b941ddafb40774105c94bb 

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

[//]: # 'The order in which to review files and what to expect when testing locally'

- Go to the settings page from the home screen
- Verify that the tabs display on the settings page and you can navigate from settings to the other tabs

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # 'Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally'

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

![Screenshot 2024-04-02 at 16 24 15 jpeg](https://github.com/calblueprint/girls-write-now/assets/156966341/72ee56cb-0da1-4e1d-babb-479028be6102)


CC: @adityapawar1

[//]: # 'This tags in Adi as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
